Quadriga/src/main/java/edu/asu/spring/quadriga/dto/ProjectBlogDTO.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.asu.spring.quadriga.dto;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ *This class represents the column mappings for projectblog table.
+ * @author Pawan Mahalle
+ */
+/**
+ * @author PawanMahalle
+ *
+ */
+@Entity
+@Table(name = "tbl_projectblog")
+@XmlRootElement
+@NamedQueries({
+    @NamedQuery(name = "ProjectBlogDTO.findAll", query = "SELECT pb FROM ProjectBlogDTO pb"),   })
+public class ProjectBlogDTO implements Serializable {
+    private static final long serialVersionUID = 1L;
+    
+    @Id
+    @Basic(optional = false)
+    @Column(name = "projectblogid")
+    private String projectBlogId;
+    
+    @Basic(optional = false)
+    @Column(name = "title")
+    private String title;
+    
+    @Basic(optional = false)
+    @Column(name = "createdDate")
+    private Date createdDate;
+    
+    @Basic(optional = false)
+    @Column(name = "projectid")
+    private String projectid;
+    
+    @Basic(optional = false)
+    @Column(name = "author")
+    private String author;
+    
+    @Lob
+    @Basic(optional = false)
```

Comment:
don't put this kind of restrictions on the database; limiting the code on this level makes it difficult to change later. This kind of logic should be handled in manager and validator classes.